### PR TITLE
Tutorial - HA Kubernetes cluster

### PR DESCRIPTION
**Pull request by DeborggraeveR**
I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Deborggraeve Randy <randy.d@live.be>

### CODE REVIEW

File: tutorials/install-kubernetes-ha-cluster/01.en.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+## Step 3 - Setup k8s-master-001
+
+### Step 3.1 - Initialize the cluster
+
+**Create the initial kubeadm config file**
+
+Make sure to set the correct certSANs domain.
+This is a dns entry pointing to the HA-Proxy public ip. This allows to run the kubectl from the local pc.
+
+`sudo nano kubeadm-config.yaml`
```
Comment (n-se) [inline tutorials/install-kubernetes-ha-cluster/01.en.md]: You now use sudo here but not above, please either use sudo everywhere necessary or dont at all.

File: tutorials/install-kubernetes-ha-cluster/01.en.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+
+`mkdir -p $HOME/.kube`
+
+`sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config`
+
+`sudo chown $(id -u):$(id -g) $HOME/.kube/config`
+
+### Step 3.2 - Copy the cluster join information
+
+Make sure u copy the kubeadm join info from the init output. We need this later.
```
Comment (n-se) [inline tutorials/install-kubernetes-ha-cluster/01.en.md]: Please do not use things like `u` for `you`

**Comment by leojakeson** [comment]
@DeborggraeveR are you still interesting in working on this? If so please let us know so we can keep this PR open.

**Comment by leojakeson** [comment]
No response from author.

**Pull request closed without merging**